app/pb_kits/playbook/pb_date_stacked/_date_stacked.jsx

```diff
@@ -36,20 +36,20 @@ const DateStacked = ({
     'reverse': reverse,
   }))
 
-  const currentYear = new Date().getFullYear().toString()
-  const dateTimestamp = new DateTime({ value: date })
-  const inputYear = dateTimestamp.toYear().toString()
+  const currentYear = moment().year()
+  const dateObject = moment(date)
+  const inputYear = dateObject.format('Y')
 
   return (
     <div className={classes}>
       <div className="pb_date_stacked_day_month">
         <Caption
-            text={dateTimestamp.toMonth().toUpperCase()}
+            text={dateObject.format('MMM').toUpperCase()}
```

Comment:
`dateTime.js` was created to prevent verbosity here. If need be, we should develop new exports/methods there.